Pipfile

```diff
@@ -24,6 +24,9 @@ textblob = "*"
 scipy = "*"
 pylint = "*"
 importlib-metadata = "*"
+language_tool_python = "*"
+language = "*"
+tool = "*"
```

Comment:
Just want to point this out in case it's being missed: can you verify both these packages are used?

Reply:
This one is added by my teammate, I think we forgot to discuss about it. These two packages will not be in used, the only package that we need in this issue is language_tool_python, I'll erase the two above.